src/en/actions.md

```diff
@@ -0,0 +1,317 @@
+# Juju Actions (DRAFT)
```

Comment:
This seems largely duplicated with charms-actions.md I don't see where this is referenced anywhere in nav or others?